numpy/core/src/umath/loops.c.src

```diff
@@ -644,6 +644,19 @@ BOOL__ones_like(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UN
 }
 
 
+/**begin repeat
+ * #kind = isnan, isinf, isfinite#
+ * #func = npy_isnan, npy_isinf, npy_isfinite#
+ * #val = NPY_FALSE, NPY_FALSE, NPY_TRUE#
+ **/
+NPY_NO_EXPORT void
+BOOL_@kind@(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
+{
+    OUTPUT_LOOP_FAST(npy_bool, *out = @val@);
```

Comment:
Why not just pass the value and move the assignment to the macro where `out` is declared?

Reply:
It was entirely to preserve calling convention to be similar to the other macros - I've implemented your suggestion in https://github.com/numpy/numpy/pull/13208